openstf/validation/validation.py

```diff
@@ -22,18 +23,22 @@
 
 
 def validate(
-    data: pd.DataFrame, flatliner_threshold: int = FLATLINER_TRESHOLD
+    pj_id: Union[int, str],
+    data: pd.DataFrame,
+    flatliner_threshold: int = FLATLINER_TRESHOLD,
 ) -> pd.DataFrame:
     logger = structlog.get_logger(__name__)
     # Drop 'false' measurements. e.g. where load appears to be constant.
     data = replace_repeated_values_with_nan(
         data, max_length=flatliner_threshold, column_name=data.columns[0]
     )
     num_const_load_values = len(data) - len(data.iloc[:, 0].dropna())
-    logger.debug(
-        f"Changed {num_const_load_values} values of constant load to NA.",
-        num_const_load_values=num_const_load_values,
-    )
+    if num_const_load_values > 0:
+        logger.warning(
```

Comment:
In general I think we should log:
* An identifier for the job (e.g. prediction job id (like you already added)
* An identifier about the type of validation/preprocessing step which was conducted
* Some (if possible generic) metrics on that validation/preprocessing step
  * Probably the number of datapoints and or the relative number of datapoints which have been modified is a good metric.